_tests.sh

```
@@ -87,131 +163,269 @@ echo "  - <test>/ok.log(all good)"
+    eval_testcase
+}
+
+# shellcheck disable=SC2317
+function 2 {
+    test="${FUNCNAME[0]}"
+    testcase_synopsis="base-repo ; clone; fetch-co : the repo has two tags and the latest is checked out"
+    testcase_header
+    {
+        cd $test
+        generate_base_repo
+        git artifact clone --url=$(pwd)/$remote_tester_repo --path $clone_tester_repo
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)